front-end/src/components/List/ListItem/BookListItem.scss

```
@@ -0,0 +1,25 @@
+@import "../../../scss/variables";
+
+.bookListItem {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: left;
+  margin-bottom: 15px;
+}
+
+.cover {
+  width: 37px;
+  height: 55px;
+  background: $color-black;
+  box-shadow: 0px 0px 6px #00000066; 
+}
+
+.text{
+  padding-left: 11px;
```

```suggestion
  padding-left: 12px;
```
We should also add this to our spacing variables.